app/database/dao/hgpt.py

```
@@ -641,133 +614,14 @@ def _convert_rgba_to_palette(png_data: bytes, target_palette_size: int) -> bytes
-        finally:
-            # 清理临时文件
-            if os.path.exists(tmp_input_path):
-                os.unlink(tmp_input_path)
-            if os.path.exists(tmp_output_path):
-                os.unlink(tmp_output_path)
+        return quantize_png(png_data, target_palette_size)
     
     @staticmethod
     def _convert_rgba_to_palette_fallback(png_data: bytes, target_palette_size: int) -> bytes:
         """
         备用的简单颜色量化（当 Pillow 不可用时）
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)